text/0000-localizing-api-messages.md

```diff
@@ -0,0 +1,135 @@
+
+- Start Date: 03-29-2022
+- RFC PR: 
+- FOLIO Issue: 
+- Submitter : Radhakrishnan Gopalakrishnan (rgopalakrishnan@ebsco.com)
+- Co-Submitter(s) : Zak Burke (zburke@ebsco.com)
+- Sub Group :
+
+# Localizing API (Backend) Messages
+
+## Summary
+  The purpose of this RFC is to define an approach that allows API developer to provide
+messages in the API response depending on the language preference set in the request. 
+
+
+## Motivation
+
+- Help users understand the information coming from the application backend using a language they understand better
+- To allow FOLIO platform to be used by users across the world 
+- To create a tremendous opportunity for growth that we can never could have achieved within just one country.
+
+### Terminology
+- Static Message - E.g. _This is something you need translate_
+- Static Message with placeholder - _You can ONLY renew {n} times_
+ 
+
+### In Scope Requirements/Use cases
+- Return localized messages based on the value passed in the accept-language header
+- Handle static messages, static messages with placeholders
+
+### Out of Scope Requirements/Use cases
+- [Plural Syntax](https://wiki.folio.org/display/I18N/How+To+translate+FOLIO#HowTotranslateFOLIO-Pluralsyntax)
+- Naming convention to be used for keys
+- Process for managing (decoupled from lokalise.com) translations
+- Returning formatted (HTML/Markdown) messages
+- Usage of [soft hyphen](https://wiki.folio.org/display/I18N/How+To+translate+FOLIO#HowTotranslateFOLIO-Softhyphentobreakwords) to break messages
+- Support customization per tenant
+- Process for back porting API/Backend messages similar to what we have for the front end 
+- Support for controlled vocabulary (runtime data/data coming from DB tables. E.g. Patron Groups for tenants)
```

Comment:
Add reference to [UXPROD-3148](https://issues.folio.org/browse/UXPROD-3148), please.